tools/provision

```diff
@@ -39,7 +37,7 @@ the Python version this command is executed with."""
     py_version = tuple(int(num) for num in py_version_list[0:2])
     venv_name = 'zulip-api-py{}-venv'.format(py_version[0])
 
-    if py_version <= (3, 1) and (not options.force):
+    if py_version <= (3, 5) and (not options.force):
```

Comment:
I grepped for 3.5 in the codebase. It's not only in this line that the version needs to be updated. @LoopThrough-i-j I think you should do the version constraint thing in a separate PR.

e.g.
```
zulip_botserver/setup.py
27:        'Programming Language :: Python :: 3.5',
32:    python_requires='>=3.5',
```

and several more.

Reply:
ok makes sense will do.

Reply:
@LoopThrough-i-j have you made the PR to drop 3.5 yet?

Reply:
#655